src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,74 @@
+package duke;
+
+import duke.command.Command;
+import duke.io.Storage;
+import duke.io.TaskList;
+import duke.parser.Parser;
+
+import java.io.File;
+
+/**
+ * This is the start of the Duke application. There 3 main task that the application can record,
+ * ToDo, Event and Deadline.
+ *
+ * @author Galvin Leow Wen Yuan
+ * @author A0200204J
+ * @version v1.0
+ */
+public class Duke {
+
+    private static final String CURRENT_DIRECTORY = System.getProperty("user.dir");
+    private final Storage storage;
+    private TaskList taskList;
+    private final Ui ui;
+
+    /**
+     * Class constructor that initialised the application.
+     * Ui, Storage and TaskList are initialised in this constructor.
+     *
+     * @param filePath path of duke.txt, duke.txt acts as a simple database.
+     */
+    public Duke(String filePath) {
+        this.ui = new Ui();
+        this.storage = new Storage(filePath);
+        try {
+            this.taskList = new TaskList(storage.load());
+        } catch (DukeException e) {
+            this.ui.showLoadingError();
+            this.taskList = new TaskList();
+        }
+    }
+
+    /**
+     * Main logic of application execution.
```

Comment:
Maybe the header comments for a method could start with a verb instead of a noun? In the coding standard, they have some suggestions like "Returns", "Sends" etc